athena/Radius.tsx

```diff
@@ -212,29 +221,35 @@ export function getPathCost(
   return !unitB || canLoad(map, unitB, unit, previousVector) ? totalCost : -1;
 }
 
+function getVisionRange(
+  map: MapData,
+  unit: Unit,
+  start: Vector,
+  radius: number,
+) {
+  const range = unit.isUnfolded()
+    ? 2
+    : unit.info.type === EntityType.Infantry &&
+        map.getTileInfo(start).type & TileTypes.Mountain
+      ? 1
+      : 0;
+  return radius + range;
+}
+
 export function visible(
   map: MapData,
   unit: Unit,
   start: Vector,
   radius: number = unit.info.configuration.vision,
 ): ReadonlyMap<Vector, RadiusItem> {
-  const vision =
-    radius +
-    (unit.isUnfolded()
-      ? 2
-      : unit.info.type === EntityType.Infantry &&
-          map.getTileInfo(start).type & TileTypes.Mountain
-        ? 1
-        : 0);
-
+  const vision = getVisionRange(map, unit, start, radius);
```

Comment:
Why did you extract this into a function?

Reply:
Readability 😅